lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -115,7 +115,12 @@ namespace Js
         // the lock for work item queue should not be locked while accessing AuxPtrs in background thread
         auto jobProcessor = this->GetScriptContext()->GetThreadContext()->GetJobProcessor();
         auto jobProcessorCS = jobProcessor->GetCriticalSection();
-        Assert(!jobProcessorCS || !jobProcessor->ProcessesInBackground() || !jobProcessorCS->IsLocked());
+#ifndef _WIN32
+        bool isLocked = false;
```

Comment:
Why is this a constant? This essentially disables the assert on xplat; if that is the intention, why not just place the `Assert` under the `#if _WIN32` ?

Reply:
No IsLocked support. Agree, we may also disable it for xplat as well